packages/swamp/src/connectors/klaviyo.ts

```diff
@@ -0,0 +1,130 @@
+import axios from 'axios';
+import { z } from 'zod';
+import { Inserts, Loader, LoaderResponse, LoaderSecrets } from '../core/types';
+
+export const Secrets = z.object({
+  apiKey: z.string(),
+});
+export type Secrets = z.infer<typeof Secrets>;
+
+export const Cursor = z.object({
+  next: z.string().optional(),
+  since: z.string(),
+});
+export type Cursor = z.infer<typeof Cursor>;
+
+type Event = {
+  id: string;
+  event: string;
+  timestamp: string;
+  [key: string]: any;
+};
+
+function getParams(cursor: Cursor) {
+  if (cursor.next) {
+    return {
+      'page[cursor]': cursor.next,
+      'filter': `greater-than(datetime,${cursor.since})`,
```

Comment:
Is there an id cursor available? This could fail if there are two records with the same datetime